Reviewer/src/main/java/com/softserveinc/reviewer/ReviewerApplication.java

```diff
@@ -27,6 +28,7 @@ public void run(final ReviewerConfiguration configuration, final Environment env
         environment.healthChecks().register("oracle", new DataProvidersHealthCheck(configuration.getOracleHealthCheckUrl()));
         environment.healthChecks().register("elasticsearch", new DataProvidersHealthCheck(configuration.getElasticSearchHealthCheckUrl()));
         environment.jersey().register(injector.getInstance(ReviewerResource.class));
+        environment.jersey().register(injector.getInstance(ReviewerStatus.class));
```

Comment:
I think we can use Healhcheck as an entry point here (instead of three existing). I'd just comment them out during that task